django_lifecycle/mixins.py

```diff
@@ -196,15 +199,15 @@ def _run_hooked_methods(self, hook: str) -> List[str]:
                 if when_field:
                     if self._check_callback_conditions(when_field, callback_specs):
                         fired.append(method.__name__)
-                        method()
+                        method(self)
```

Comment:
The flip side of that thing I mentioned earlier, it's returning functions not bound methods, so we can't rely on internal Python magic to pass ```self```.  But functionally it's the same thing.